2022-07_sprint/sprint_parties/index.md

```diff
@@ -1,8 +1,185 @@
 # Sprint parties
 ## Organizers
+### Data Umbrella Organizers
+
+::::{grid} 2 2 3 5
+:gutter: 3
+
+:::{grid-item-card}
+:link: https://reshamas.github.io
+:shadow: none
+:class-body: sd-text-center
+:class-card: border-0
+
+<img alt="Reshama Shaikh" src="../../_static/reshama.jpg" class="rounded-circle">
+
+Reshama Shaikh
+:::
+:::{grid-item-card}
+:link: https://twitter.com/BerylKanali
+:shadow: none
+:class-body: sd-text-center
+:class-card: border-0
+
+<img alt="Beryl Kanali" src="../../_static/beryl.JPG" class="rounded-circle">
+
+Beryl Kanali
+:::
+:::{grid-item-card}
+:link: https://github.com/symeneses
+:shadow: none
+:class-body: sd-text-center
+:class-card: border-0
+
+<img alt="Sandra Meneses" src="../../_static/sandra.jpg" class="rounded-circle">
+
+Sandra Meneses
```

Comment:
@symeneses is github ok as the link your image points to?

You also need to add a `sandra.jpg` or png to the `_static` folder or provide a link to a raw stable image.

Reply:
@OriolAbril 
For @symeneses:
primary is LI:  https://www.linkedin.com/in/symeneses/
secondary is GH:  https://github.com/symeneses

Reply:
I will add the image in the folder.

Reply:
@symeneses I added your image to the folder already.